src/components/RangeSlider/utilities/index.ts

```diff
@@ -1,7 +1,7 @@
 export const CSS_VAR_PREFIX = '--Polaris-RangeSlider-';
-export const DEFAULT_RANGE_SLIDER_PROPS = {
-  min: 0,
-  max: 100,
-  step: 1,
-};
+export enum RangeSliderDefault {
```

Comment:
Just curious about the rationale behind choosing screaming snake case vs not?

Reply:
Took a quick look at our [naming guidelines](https://github.com/Shopify/javascript#2.2), and PascalCase is what we use for enums.